docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -243,9 +242,9 @@ If you're a library author and want to be extra safe, check that consumers can w
 
 **When do the changes take effect?**
 
-* In 1.8.20, usage of these properties provokes a warning
-* In 1.9.20, usage of these properties provokes an error
-* In 2.0, support for these properties will be removed, and Kotlin Gradle Plugin will ignore them 
+* 1.8.20: report a warning when these properties are used
```

Comment:
We already used this approach in the Multiplatform compatibility guide when we have a list for deprecation cycles. Now it's consistent in all sections

Here we follow Kotlin compatibility guides that have described the cycle in this way since [Kotlin 1.4](https://kotlinlang.org/docs/compatibility-guide-14.html)

Reply:
Well from the link the difference is quite clear. The linked page (and any of the language compatibility pages) doesn't give any single recommendation to users, like "Migrate your code by replacing...". And KMP Compatibility page does that all over the place.

Which one is the proper way is an open question, but it's clear that decision from Language Compatibility Guide can't be blindly retrofitted to KMP Compatibiltiy Guide. They're written in significantly different styles.

Reply:
But these are different sections. We give recommendations to users in the **What's the best practice now?** section, and the **When do the changes take effect?** section just lists Kotlin versions that describe the deprecation cycle as stated in the header, which is exactly the same as in Kotlin compatibility guides 

added a sentence about the deprecation cycle

Reply:
Ok